Storage/Storage/Model/WooCommerceFeature.swift

```diff
@@ -0,0 +1,12 @@
+public struct WooCommerceFeature: Codable, Identifiable {
```

Comment:
I think this should be just called `Feature`, why did you choose to prepend `Woocommerce` to it?